Sources/GRPCCore/Documentation.docc/Tutorials/Hello-World/Hello-World.tutorial

```diff
@@ -0,0 +1,126 @@
+    a gRPC server. You'll also see how to create a client and use it to call the server.
+
+    The tutorial assumes you are comfortable with both Swift and the basic concepts of gRPC. If you
+    aren't then check out the Swift [Getting Started](https://www.swift.org/getting-started/) guide
+    and the [gRPC website](https://grpc.io) for more information.
+
+    You'll need a local copy of the example code to work through this tutorial. Download the example
+    code from our GitHub repository if you haven't done so already. You can do this by cloning the
+    repository by running the following command in a terminal:
+
+    ```console
+    git clone https://github.com/grpc/grpc-swift
+    ```
+
+    The rest of the tutorial assumes that your current working directory is the cloned `grpc-swift`
+    directory.
+  }
+
+  @Section(title: "Run a gRPC application") {
+    Let's start by running the existing Greeter application.
+
+    @Steps {
+      @Step {
+        In a terminal run `swift run hello-world serve` to start the server. By default it'll start
+        listening on port 31415.
+
+        @Code(name: "Console.txt", file: "hello-world-sec02-step01.txt")
+      }
+
+      @Step {
+        In another terminal run `swift run hello-world greet` to create a client, connect
+        to the server you started and send it a request and print the response.
+
+        @Code(name: "Console.txt", file: "hello-world-sec02-step02.txt")
+      }
+
+      @Step {
+        Congratulations! You've just run a client-server application with gRPC Swift. You can now
+        cancel the two running processes.
+      }
+    }
+  }
+
+  @Section(title: "Update a gRPC service") {
+    Now let's look at how to update the application with an extra method on the server for the
+    client to call. Our gRPC service is defined using protocol buffers; you can find out lots more
+    about how to define a service in a `.proto` file in [What is gRPC?](https://grpc.io/docs/what-is-grpc/).
+    For now all you need to know is that both the server and client "stub" have a `SayHello` RPC
+    method that takes a `HelloRequest` parameter from the client and returns a `HelloReply` from
+    the server.
+
+    @Steps {
+      @Step {
+        Open `HelloWorld.proto` in the `Sources/Examples/v2/hello-world` directory to see how the
+        service is defined.
+
+        @Code(name: "HelloWorld.proto", file: "hello-world-sec03-step01.proto")
+      }
+
+      @Step {
+        Let's update it so that the `Greeter` service has two methods. Add a  new `SayHelloAgain`
+        method, with the same request and response types.
+
+        @Code(name: "HelloWorld.proto", file: "hello-world-sec03-step02.proto")
+      }
+    }
+  }
+
+  @Section(title: "Update and run the application") {
+    You need to regenerate the stubs as the service definition has changed. You'll learn how to do
+    this in another tutorial.
```

Comment:
I'm a bit confused that you mention "You'll learn how to do this in another tutorial." but then proceed to say how to update/regenerate the stubs.

Also, do we want to mention the SPM plugin?

Reply:
Yeah, this isn't clear at all. Looks like I half rewrote it. The point I was trying to make here is that the reader needs to update the stubs and to do that they should just to run our script. Learning how to do it properly will be done in another tutorial as I want to keep this as simple as possible and I want to avoid duplicating all of that information across tutorials.

I'll rewrite this part.